code/datums/ai/ai_law_sets.dm

```diff
@@ -83,7 +83,6 @@
 /datum/ai_laws/drone/New()
 	add_inherent_law("Preserve, repair and improve the station to the best of your abilities.")
 	add_inherent_law("Cause no harm to the station or anything on it.")
```

Comment:
Same comment as above (changing "station" for "colony"), although we should add an exception for self defense as drones can and **will** get attacked by roaches and spiders...unless they're meant to just run away from them